SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/packages.config

```diff
@@ -1,15 +1,43 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Xamarin.Build.Download" version="0.4.9" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Arch.Core.Common" version="1.1.1.1" targetFramework="monoandroid90" />
```

Comment:
Please try updating in demo with build nugets from CI to see if we need nuspec update to help conflicts.

Reply:
https://portal-server-core-integration.dev.avalanch.es/orgs/appcenter-sdk/apps/xamarin-puppet-android-03/build/branches/fix%2Fupdate-depends/builds/343